website/content/api-docs/system/inspect/router.mdx

```diff
@@ -0,0 +1,97 @@
+---
+layout: api
+page_title: /sys/internal/inspect/router - HTTP API
+description: >-
+  The '/sys/internal/inspect/router' endpoint focuses on viewing the contents of specific structures in the internal router subsystem.
+---
+
+# `/sys/raw`
```

Comment:
```suggestion
# `/sys/internal/inspect/router`
```

Reply:
Might be worth grepping for "raw" on these new pages, though I didn't see it anywhere else.